lib/git_reflow/git_server/git_hub/pull_request.rb

```diff
@@ -78,6 +78,45 @@ def approved?
           end
         end
 
+        def merge_feature_branch(feature_branch_name, options = {})
```

Comment:
I would still like to see these named `merge!`, as we can infer what the feature branch name is (`current_branch`), and the bang should be present so the user knows that it is dangerous (will raise an error).  Also, since we're now in the context of the pull request, the only option we'd need to look for is a `destination_branch` override.  Every other option can pull on information from the instance of itself.  I think it would make sense to have a new method here as well that generates the commit message so that we can slim down the complexity of this method.